client/src/components/Ticketing/ticketingmanager/ticketing/ticketingSlice.ts

```diff
@@ -356,8 +489,26 @@ export const selectCartTicketCount = (state: RootState): {[key: number]: number}
       }
       , {},
   );
+export const getNumTickets = (state: RootState): {[key: number]: number} =>
+  state.ticketing.cart.reduce(
+      /*
+        (acc, item) => {
+          acc + item.qty;
+        }, {},
+      */
```

Comment:
This as well?

Reply:
Also part of next step. It will be gone.